Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:latest
+FROM alpine:latest as builder
```

Comment:
Do we really need to do a builder image? Is that to get the certs passed on to the scratch image? Not against it just trying to understand!

Reply:
yeah, I believe so...
You could argue that `builder` is a bit misleading since we're not actually building it in the container?

Maybe `FROM alpine:latest as certs`?